src/ccnl-core/include/ccnl-dump.h

```diff
@@ -15,33 +15,33 @@
  * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
  * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
- *
- * File history:
- * 2012-05-06 created
- * 2013-10-21 extended for crypto <christopher.scherb@unibas.ch>
- * 2017-26-06 adapted to ccnl v2
  */
 
 #ifndef CCNL_DUMP_H
 #define CCNL_DUMP_H
 
-//#ifdef USE_DEBUG
+/**
+ * @brief A macro which prints \p n  whitespace characters
```

Comment:
for consistency I would stick to `@p` and `@ref` instead of `\p` and `\ref

Reply:
I'll change it